src/main/resources/locales/en/cst-and-pht-weekly-hearing-list.json

```diff
@@ -0,0 +1,12 @@
+{
+    "date": "Date",
+    "caseName": "Case name",
+    "hearingLength": "Hearing length",
+    "hearingType": "Hearing type",
+    "venue": "Venue",
+    "additionalInformation": "Additional Information",
```

Comment:
Can we make the casing "Information" lower case to match up with the other header?

Reply:
Updated